lib/human_player.rb

```diff
@@ -1,12 +1,18 @@
-class HumanPlayer
+class Human_player
   attr_reader :id, :mark
-  def initialize(id, mark, inOut)
-    @inOut = inOut
+  def initialize(id, mark, in_out)
+    @in_out = in_out
     @id = id
     @mark = mark
   end
 
-  def make_move(max_turns)
-    @inOut.select_move
+  def make_move(board)
```

Comment:
Is this because the Computer Player needs a board?
If so, is there another way for only the computer to have access to the board, without having to pass the board instance to the human player as well?